.github/workflows/motion-benchmarks-comment.yml

```diff
@@ -3,15 +3,14 @@ name: 'Motion Benchmark Comment on PR'
 on:
   workflow_run:
     workflows: ["Motion Pull Request Update"]
-    types:
-      - completed
+    types: [ completed ]
 
 jobs:
   motion_comment:
     name: 'Post Comment on PR'
     runs-on: [x64, qemu-host]
     container:
-      image: ghcr.io/viamrobotics/canon:amd64-cache
+      image: ghcr.io/viamrobotics/canon:amd64
```

Comment:
It probably doesn't actually matter here either way, but if it's not building Go code, the cache is useless. This could actually be ubuntu-latest and not need our infra actually. Will change.